delia/readers/image/dicom_reader.py

```diff
@@ -231,7 +240,7 @@ def get_dicom_headers(self, remove_segmentations: bool = False) -> List[pydicom.
                 _logger.info(f"Patient ID : {series_data.dicom_header.PatientID}")
                 _logger.debug(f"Path to images folder : {self._path_to_patient_folder}")
 
-            _logger.debug(f"  Series description : {series_data.series_description}")
+            _logger.debug(f"  Tag value : {series_data.tag_value}")
```

Comment:
Use `self.tag` instead of `Tag value`, so the user can confirm that the right tag is used.